### PR TITLE
add Product Cards solution

### PR DESCRIPTION
**Pull request by TamaraShukalo**
- [DEMO LINK](https://TamaraShukalo.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://TamaraShukalo.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    color: $text-color-main-accent;
+    font-style: normal;
```
Comment (tetiana-ivanchenko) [inline src/styles/blocks/card.scss:17]: it's normal by default (redundant)

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $text-color-secondary;
+    font-style: normal;
+    font-weight: 400;
```
Comment (tetiana-ivanchenko) [inline src/styles/blocks/card.scss:27]: font-style: normal and  font-weight: 400 is default values, so it's declaration is redundant

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,61 @@
...
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__review-box">
+        <div class="stars stars--4">
```
Comment (tetiana-ivanchenko) [inline src/index.html:32]:
```suggestion
      <div class="card__review-box">
        <div class="card__stars stars stars--4">
```

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    border-radius: 5px;
+    border: 1px solid $color-blue-accent;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__buy-button:hover {
+    color: $color-blue-accent;
+    background-color: $color-white;
+  }
```
Comment (anstsot) [inline src/styles/blocks/card.scss:81]: use the power of nesting
```suggestion

    &:hover {
      color: $color-blue-accent;
      background-color: $color-white;
    }
  }
```

**Review by anstsot — APPROVED** [review]
Good work, approving:sparkles:

**Pull request closed without merging**